vtex-sites/storecomponents.store
# Migrate Carousel component

## What's the purpose of this pull request?

This replaces the deprecate Carousel being used in the Home page with the new Carousel component added to `@vtex/store-ui` by https://github.com/vtex/faststore/pull/815.

## How to test it?

Just check out the deploy preview below, the carousel should be the first component you see below the menu.
